.github/workflows/rpm-build-and-test.yml

```diff
@@ -0,0 +1,104 @@
+name: RPM Build and Test
+
+env:
+  # TODO: Surely this is a GH variable of some sort for the name of the repo
+  NAME: argobots
```

Comment:
Not tested but some googling says this should work
```
"${{github.repository}}" | cut -d "/" -f 2
```

And this, but NOT for PRs unfortunately
```
${{ github.event.repository.name }}
```

Reply:
I knew there had to be one but just feeling the time is running out to get mired in the small details.

Great suggestion though.  If I feel like the time will allow, I'll push an update for that.

Reply:
Agree. No need to stress on this